tox.ini

```diff
@@ -16,12 +17,14 @@ deps=
     # isn't compatible with the older versions of numpy
     # that we test with.
     legacy: gwcs==0.9.1
+# NOTE: this dependency on `six` is only here because `gwcs <= 0.18.1` requires it
+    legacy: six~=1.4.0
     legacy: semantic_version==2.8
     legacy: pyyaml==3.13
     legacy: jsonschema==4.0.1
-    legacy: numpy~=1.14.6
-    legacy: pytest~=4.6.11
-    legacy: astropy~=3.0.0
+    legacy: numpy~=1.18
+    legacy: pytest~=6.0.0
+    legacy: astropy~=5.0.4
```

Comment:
`pytest` was updated so that it can read `pyproject.toml`, `astropy` was updated to reflect the current minimum supported version, and `numpy` was updated to match `astropy`'s new version